Doc/library/string.rst

```diff
@@ -312,12 +312,13 @@ non-empty format specification typically modifies the result.
 The general form of a *standard format specifier* is:
 
 .. productionlist:: format-spec
-   format_spec: [[`fill`]`align`][`sign`]["z"]["#"]["0"][`width`][`grouping_option`]["." `precision`][`type`]
+   format_spec: [[`fill`]`align`][`sign`]["z"]["#"]["0"][`width`][`grouping_option`]["." `precision` [`fraction_grouping`]][`type`]
    fill: <any character>
    align: "<" | ">" | "=" | "^"
    sign: "+" | "-" | " "
    width: `~python-grammar:digit`+
    grouping_option: "_" | ","
+   fraction_grouping: "_"
```

Comment:
What's the reasoning behind not supporting `.,f`? Simplicity?
I feel like it's easier to understand if comma's are supported in both positions.

Reply:
You can think of it like `%f`. Like this: `%.1f`, `%2.1f` ......

Reply:
> I'm from country where comma used as a decimal point

Me too, but if we make it consistent we don't need a "but..." when explaining the feature:

> You can now separate thousands in the fractional part, **but you can't use a comma as separator**.

Another advantage is that we can re-use `grouping_option` in the grammar.